kiosk/kiosk_browser/connection.py

```diff
@@ -6,20 +10,23 @@
 
 check_connection_url = 'http://captive.dividat.com/'
 sleep_time_not_connected = 5
-sleep_time_connected = 60
+sleep_time_connected = 6
+sleep_time_proxy = 6
 
 class Status(Enum):
     DISCONNECTED = auto()
     CAPTIVE = auto()
     CONNECTED = auto()
+    PROXY = auto()
```

Comment:
I find the blurring between the status a bit confusing. When behind a proxy we can still be connected or disconnected. Would it make sense to prefix the non-proxy status with `DIRECT_` to hint at them being a "world of their own".

Reply:
That’s right. Prefixing with `DIRECT_` and adding a comment should be enough, done in https://github.com/dividat/playos/pull/54/commits/e1293840cadf356ef5f45c0f53c68eb21e38b58a.